xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/accounts/AccountTransactionsRequestParams.java

```diff
@@ -28,20 +29,56 @@
 @Value.Immutable
 @JsonSerialize(as = ImmutableAccountTransactionsRequestParams.class)
 @JsonDeserialize(
-  as = ImmutableAccountTransactionsRequestParams.class,
-  using = AccountTransactionsRequestParamsDeserializer.class
+    as = ImmutableAccountTransactionsRequestParams.class,
+    using = AccountTransactionsRequestParamsDeserializer.class
 )
 public interface AccountTransactionsRequestParams extends XrplRequestParams {
 
   /**
    * Construct a builder for this class.
    *
    * @return An {@link ImmutableAccountTransactionsRequestParams.Builder}.
+   * @deprecated Prefer one of the other defined builders so that you are overtly
+   *     specifying which ledger index or type you want.
    */
+  @Deprecated
   static ImmutableAccountTransactionsRequestParams.Builder builder() {
     return ImmutableAccountTransactionsRequestParams.builder();
   }
 
+  /**
+   * Construct a builder for this class with {@link LedgerSpecifier} field set.
+   *
+   * @param ledgerSpecifier A specifier of type {@link LedgerSpecifier} to specify a ledger.
+   *
+   * @return An {@link ImmutableAccountTransactionsRequestParams.Builder}.
+   */
+  static ImmutableAccountTransactionsRequestParams.Builder builder(LedgerSpecifier ledgerSpecifier) {
+    return ImmutableAccountTransactionsRequestParams.builder().ledgerSpecifier(Optional.ofNullable(ledgerSpecifier));
+  }
+
+  /**
+   * Construct a builder for this class with ledgerIndexRange field set.
+   *
+   * @param ledgerIndexMinimum A lower bound of ledgerIndex of type {@link LedgerIndexBound}.
+   * @param ledgerIndexMaximum An upper bound of ledgerIndex of type {@link LedgerIndexBound}.
+   *
+   * @return An {@link ImmutableAccountTransactionsRequestParams.Builder}.
+   */
+  static ImmutableAccountTransactionsRequestParams.Builder builder(
+      LedgerIndexBound ledgerIndexMinimum,
+      LedgerIndexBound ledgerIndexMaximum) {
+    LedgerIndexBound ledgerIndexMin = ledgerIndexMinimum;
+    LedgerIndexBound ledgerIndexMax = ledgerIndexMaximum;
```

Comment:
Why are these needed? Not sure if I'm missing something...

Reply:
no, these were for the old == null logic. Taking these off.